### PR TITLE
Add Community Profile

### PR DESCRIPTION
**Pull request by fbernutz**
💁‍♀️✨

### CODE REVIEW

**Comment by zntfdr** [comment]
pineapple on pizza 👀

**Comment by zntfdr** [comment]
Welcome to the community 🥳

File: content/community/felibe444.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Felizia Bernutz
+twitter: felibe444
+github: fbernutz
+website: https://fbernutz.github.io/
+---
+
+iOS dev • sketchnoting • accessibility • pineapple on pizza • longboarding • she/her
```
Comment (zntfdr) [inline content/community/felibe444.md:8]: pineapple on pizza 👀

Comment (fbernutz) [reply content/community/felibe444.md:8]: best pizza 🍍

**Pull request merged**